src/content/about/metrics.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Metrics
+weight: 4
+---
+
+- [Worker Script Metrics](#worker-script-metrics)
+  - [Invocation statuses](#invocation-statuses)
+  - [Requests](#requests)
+  - [CPU time](#cpu-time)
+- [Zone Metrics](#zone-metrics)
+  - [Total requests](#total-requests)
+  - [Status codes](#status-codes)
+  - [Bandwidth](#bandwidth)
+  - [Subrequests](#subrequests)
+- [GraphQL](#graphql)
+
+Workers metrics show performance and usage of your Workers and help diagnose issues.
+
+Workers Metrics are available in Zone and Worker script contexts.
+
+## Worker Script Metrics
```

Comment:
```suggestion
```

Title will already be rendered

Reply:
Can you explain more about this one?

Reply:
Hmm, here's what I see locally.  
<img width="673" alt="Screen Shot 2020-04-23 at 1 22 09 PM" src="https://user-images.githubusercontent.com/2585828/80135249-98cd3d80-8565-11ea-96ea-9c5cf741531e.png">

Reply:
"Metrics" and "Worker Script Metrics" are the synonymous in the context of Workers docs.

Reply:
I don't see how they're synonymous.  Metrics is the name of the feature and Worker Script Metrics is the part of the feature pertaining to individual worker.  The other part is Zone Metrics.